joomla/joomla-cms
# [4.0] Plugin for managing Atum settings

### Summary of Changes

Currently Atum params are hardcoded in com_users form. Unless we expect every other template to follow the same settings, this is totally wrong.

This moves those settings to a separate plugin.

### Testing Instructions

Edit your profile in backend, change some Atum settings.
Check that they're still respected.

### Expected result

Works like before.

### Documentation Changes Required

IDK

### TODO
- [x] SQL files
